frontend/src/js/external-forms/FormConfigLoader.tsx

```diff
@@ -124,7 +124,7 @@ const FormConfigLoader: FC<Props> = ({
         // from string, e.g. 'next'
         // to SelectValueT, e.g. { value: 'next', label: 'Next' }
         const field = collectAllFormFields(formConfig.fields).find(
-          (f) => f.type !== "GROUP" && f.name === fieldname,
+          (f): f is Field | Tabs => f.type !== "GROUP" && f.name === fieldname,
```

Comment:
I'm a  bit mystified by the `(f): f is Field | Tabs: ...` notation. Does this cause `field` to be Field | Tab without actually specifying the return type of find (which I expect to take an arrow function to boolean?)

Reply:
That's called a "type guard", we have quite some of those in our helper methods